dcat/index.html

```diff
@@ -3387,6 +3387,225 @@ <h2>Resource life-cycle</h2>
     
 </section>
 
+<section class="informative" id="dataset-series">
+
+<h2>Dataset series</h2>
+
+<div class="issue" data-number="868"></div>
+
+<p>With "dataset series" we refer to data, somehow interrelated, that are published separately, although they could be  merged into a single dataset. An example is budget data split by year and/or country, instead of being made available in a single dataset.</p>
+
+<p>Dataset series are defined in [[ISO-19115]] as a <q>collection of datasets [&hellip;] sharing common characteristics</q>. However, their use is not limited to geospatial data, although in other domains they can be named differently (e.g., time series, data slices) and defined more or less strictly (see, e.g., the notion of "dataset slice" in [[VOCAB-DATA-CUBE]]).</p>
+
+<p>The reasons and criteria for splitting data into series are manyfold, and they may be related to, e.g., data characteristics, publishing process, and how they are typically used. For instance, data huge in size (as geospatial ones) are more easily handled (by data providers as well as data consumers) by splitting them in smaller ones. Another example is data that are released on a yearly basis, which are typically published as separate datasets, instead of appending the new data to the first in the series.</p>
```

Comment:
@andrea-perego: perhaps I would revise it as in the following, but your mastery of English is sure better than mine ;)
```suggestion
<p>The reasons and criteria for splitting data into series are manyfold, and they may be related to, e.g., data characteristics, publishing process, and how they are typically used. For instance, data huge in size (as geospatial ones) are more easily handled (by data providers as well as data consumers) by splitting them into smaller ones. Another example is data released on a yearly basis, which are typically published as separate datasets, instead of appending the new data to the first in the series.</p>
```